packages/apps/frontera/src/routes/src/components/CommandMenu/commands/ContactBulkCommands.tsx

```diff
@@ -0,0 +1,81 @@
+import { observer } from 'mobx-react-lite';
+
+import { Tag01 } from '@ui/media/icons/Tag01';
+import { Clock } from '@ui/media/icons/Clock';
+import { useStore } from '@shared/hooks/useStore';
+import { Certificate02 } from '@ui/media/icons/Certificate02';
+import { Command, CommandItem, CommandInput } from '@ui/overlay/CommandMenu';
+import { GlobalSharedCommands } from '@shared/components/CommandMenu/commands/GlobalHub.tsx';
+
+export const ContactBulkCommands = observer(() => {
+  const store = useStore();
+  const selectedIds = store.ui.commandMenu.context.ids;
+  const label = `${selectedIds?.length} contacts`;
+
+  return (
+    <Command>
+      <CommandInput label={label} placeholder='Type a command or search' />
+      <Command.List>
```

Comment:
Rebase onto otter and wrap this component in `CommandsContainer` component to include search orgs and contacts